compiler/rustc_target/src/spec/targets/aarch64_unknown_fuchsia.rs

```diff
@@ -1,6 +1,24 @@
-use crate::spec::{SanitizerSet, StackProbeType, Target, TargetOptions, base};
+use crate::spec::{Cc, LinkerFlavor, Lld, SanitizerSet, StackProbeType, Target, base};
 
 pub(crate) fn target() -> Target {
+    let mut base = base::fuchsia::opts();
+    base.features = "+v8a,+crc,+aes,+sha2".into();
```

Comment:
While NEON is mandatory for AArch64, other targets appear to be setting it explicitly.
```suggestion
    base.features = "+v8a,+crc,+aes,+sha2,+neon".into();
```

Reply:
done